src/components/forms/PSESponsorshipsForm.tsx

```diff
@@ -559,679 +507,1118 @@ export const PSESponsorshipsForm: FC = () => {
               _placeholder={{ fontSize: 'input' }}
               color='brand.paragraph'
               fontSize='input'
-              mt={3}
-              {...register('eventName', {
-                required: true,
-                maxLength: 255
+              {...register('contactTelegram', {
+                maxLength: 150
               })}
             />
 
-            {errors?.eventName?.type === 'required' && (
-              <Box mt={1}>
-                <PageText as='small' fontSize='helpText' color='red.500'>
-                  Event name is required.
-                </PageText>
-              </Box>
-            )}
-            {errors?.eventName?.type === 'maxLength' && (
+            {errors?.contactTelegram?.type === 'maxLength' && (
               <Box mt={1}>
                 <PageText as='small' fontSize='helpText' color='red.500'>
-                  Event name cannot exceed 255 characters.
+                  Contact Telegram handle cannot exceed 150 characters.
                 </PageText>
               </Box>
             )}
           </FormControl>
 
-          <FormControl id='event-date-control' isRequired mb={8}>
-            <FormLabel htmlFor='eventDate' mb={1}>
-              <PageText display='inline' fontSize='input'>
-                Event date
-              </PageText>
-            </FormLabel>
+          <Controller
+            name='category'
+            control={control}
+            defaultValue={{ value: '', label: '' }}
+            render={({ field: { onChange }, fieldState: { error } }) => (
```

Comment:
Missing required check for this field.
```suggestion
            defaultValue={{ value: '', label: '' }}
            rules={{ required: true, validate: selected => selected.value !== '' }}
            render={({ field: { onChange }, fieldState: { error } }) => (
```